src/sass/lightbox-dialog/lightbox-dialog.scss

```diff
@@ -192,6 +193,41 @@ button.icon-btn.lightbox-dialog__prev {
         transform: rotate(180deg);
     }
 }
+
+.lightbox-dialog--fullscreen .lightbox-dialog__window,
+.lightbox-dialog--large .lightbox-dialog__window {
+    align-self: center;
+    margin: var(--spacing-100);
```

Comment:
Something's slightly off on the modal. Expanded appears to have equal scrim exposure all around. 

<kbd><img width="481" alt="image" src="https://github.com/eBay/skin/assets/1675667/bfaa61d8-81ed-428d-99c4-e577015c4b9f"></kbd>

The large modal seems to have less space at the top and bottom. 

<kbd><img width="504" alt="image" src="https://github.com/eBay/skin/assets/1675667/3cfaf709-1578-4e4a-aeca-b829d839ef85"></kbd>

Reply:
Fixed